simulations/vip-xxx/bscmainnet.ts

```diff
@@ -0,0 +1,7 @@
+import { forking, testVip } from "src/vip-framework";
+
+import vip599 from "../../vips/vip-xxx/bscmainnet";
+
+forking(52815412, async () => {
+  testVip("vip-599", await vip599(), {});
+});
```

Comment:
add Pre and Post VIP simulations to check the new facets have been added as well as their respective new function selectors .